Unable to build the latest version (`0.0.1a3`)

### Solution to issue cannot be found in the documentation.

- [x] I checked the documentation.

### Issue

Here are the logs:

```
2024-12-18T09:31:35.6691048Z ERROR:conda.stderr:
2024-12-18T09:31:35.6691399Z ClobberWarning: Conda was asked to clobber an existing path.
2024-12-18T09:31:35.6691853Z   source path: /home/conda/feedstock_root/build_artifacts/pkg_cache/pdfminer.six-20240706-pyhd8ed1ab_0/site-packages/pdfminer/utils.py
2024-12-18T09:31:35.6692561Z   target path: /home/conda/feedstock_root/build_artifacts/markitdown_1734514163518/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.10/site-packages/pdfminer/utils.py
2024-12-18T09:31:35.6693122Z 
2024-12-18T09:31:35.6693295Z 
2024-12-18T09:31:35.6693450Z 
2024-12-18T09:31:40.1254276Z done
2024-12-18T09:31:40.1643874Z export PREFIX=/home/conda/feedstock_root/build_artifacts/markitdown_1734514163518/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place
2024-12-18T09:31:40.1683920Z export SRC_DIR=/home/conda/feedstock_root/build_artifacts/markitdown_1734514163518/test_tmp
2024-12-18T09:31:41.3779862Z import: 'markitdown'
2024-12-18T09:31:42.0746950Z import: 'markitdown'
2024-12-18T09:31:42.1880455Z + pip check
2024-12-18T09:31:42.5846778Z markitdown 0.0.1a3 requires openai, which is not installed.
2024-12-18T09:31:44.1778718Z WARNING: Tests failed for markitdown-0.0.1a3-pyh9f4841b_0.conda - moving package to /home/conda/feedstock_root/build_artifacts/broken
2024-12-18T09:31:44.2179798Z Traceback (most recent call last):
2024-12-18T09:31:44.2180936Z   File "/opt/conda/lib/python3.12/site-packages/conda_build/build.py", line 3481, in test
2024-12-18T09:31:44.2196568Z     utils.check_call_env(
2024-12-18T09:31:44.2197457Z   File "/opt/conda/lib/python3.12/site-packages/conda_build/utils.py", line 406, in check_call_env
2024-12-18T09:31:44.2201048Z     return _func_defaulting_env_to_os_environ("call", *popenargs, **kwargs)
2024-12-18T09:31:44.2203567Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-12-18T09:31:44.2204415Z   File "/opt/conda/lib/python3.12/site-packages/conda_build/utils.py", line 382, in _func_defaulting_env_to_os_environ
2024-12-18T09:31:44.2205180Z     raise subprocess.CalledProcessError(proc.returncode, _args)
2024-12-18T09:31:44.2205878Z subprocess.CalledProcessError: Command '['/bin/bash', '-o', 'errexit', '/home/conda/feedstock_root/build_artifacts/markitdown_1734514163518/test_tmp/conda_test_runner.sh']' returned non-zero exit status 1.
2024-12-18T09:31:44.2206452Z 
2024-12-18T09:31:44.2208361Z During handling of the above exception, another exception occurred:
2024-12-18T09:31:44.2208816Z 
2024-12-18T09:31:44.2210585Z Traceback (most recent call last):
2024-12-18T09:31:44.2211368Z   File "/opt/conda/bin/conda-build", line 11, in <module>
2024-12-18T09:31:44.2211972Z     sys.exit(execute())
2024-12-18T09:31:44.2212374Z              ^^^^^^^^^
2024-12-18T09:31:44.2213018Z   File "/opt/conda/lib/python3.12/site-packages/conda_build/cli/main_build.py", line 648, in execute
2024-12-18T09:31:44.2213935Z     api.build(
2024-12-18T09:31:44.2214661Z   File "/opt/conda/lib/python3.12/site-packages/conda_build/api.py", line 211, in build
2024-12-18T09:31:44.2215024Z     return build_tree(
2024-12-18T09:31:44.2215325Z            ^^^^^^^^^^^
2024-12-18T09:31:44.2215643Z   File "/opt/conda/lib/python3.12/site-packages/conda_build/build.py", line 3668, in build_tree
2024-12-18T09:31:44.2220387Z     test(pkg, config=metadata.config.copy(), stats=stats)
2024-12-18T09:31:44.2223747Z   File "/opt/conda/lib/python3.12/site-packages/conda_build/build.py", line 3495, in test
2024-12-18T09:31:44.2224870Z     tests_failed(
2024-12-18T09:31:44.2225509Z   File "/opt/conda/lib/python3.12/site-packages/conda_build/build.py", line 3542, in tests_failed
2024-12-18T09:31:44.2231350Z     raise CondaBuildUserError("TESTS FAILED: " + os.path.basename(pkg))
2024-12-18T09:31:44.2232174Z conda_build.exceptions.CondaBuildUserError: TESTS FAILED: markitdown-0.0.1a3-pyh9f4841b_0.conda
2024-12-18T09:31:53.2181306Z 
2024-12-18T09:31:53.2232118Z ##[error]Bash exited with code '1'.
2024-12-18T09:31:53.2285360Z ##[section]Finishing: Run docker build
```

### Installed packages

```shell
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1113873&view=logs&j=7b6f2c87-f3a7-5133-8d84-7c03a75d9dfc&t=9eb77fd2-8ddd-5444-8fc0-71cb28dcb736
```

### Environment info

```shell
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1113873&view=logs&j=7b6f2c87-f3a7-5133-8d84-7c03a75d9dfc&t=9eb77fd2-8ddd-5444-8fc0-71cb28dcb736
```

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -15,7 +15,7 @@ build:
     - markitdown = markitdown.__main__:main
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 0
+  number: 1
 
 requirements:
   host:

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -39,6 +39,7 @@ requirements:
     - speechrecognition
     - pathvalidate
     - pdfminer-six
+    - openai
 
 test:
   imports:

